manual/git-disable-gcm/git-disable-gcm.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>git-disable-gcm</id>
-    <title>Disable Git Credential Manager</title>
+    <title>[DEPRECATED] Disable Git Credential Manager</title>
```

Comment:
I know, we had this discussion already. I'll put similar packages in the future in another repository.